doc/5.reference/expr-help.pd

```diff
@@ -1,4 +1,4 @@
-#N canvas 275 90 942 546 12;
```

Comment:
the changes for float-help look good enough (but i don't understand why you changed `[value x]` to `[value f]`)

Reply:
trying to track back my thought process, I suspect I first copied [value x] to the help file of [int], and then thought it was a good idea to have different variable names. Then it seems I ended up changing both